scout/constants/variant_tags.py

```diff
@@ -229,6 +229,24 @@
         }
 }
 
+CANCER_SPECIFIC_VARIANT_DISMISS_OPTIONS = {
+    44: {
+        'label': 'Possible Germline',
+        'description': 'Variant is possibly a germline event.',
+        'evidence': []
+        },
+    45: {
+        'label': 'Low count normal',
+        'description': 'If normal sample exists, variant has too few reads in normal sample "AD".',
```

Comment:
I can't follow this one: how about just
```variant has too few reads in normal sample "AD".```.

Reply:
Ok! I don't know where that description comes from .. Probably you, a few months ago! :)

Reply:
I'm sure! 😆